### PR TITLE
refactor: remove plugins from subgraph

### PR DESCRIPTION
**Pull request by jordaniza**
## Description

This PR merges as follows:

```mermaid
flowchart LR
    fsplit-subgraph-remove-plugins("f/split-subgraph-remove-plugins") --> fsplit_subgraph("f/split-subgraph") --> fsplit("f/split") --> develop
    other_prs("{other prs}") --> fsplit_subgraph
```

We are staging these PRs to make reviewing and readability easier but all achieve the same goal: deprecating plugins in OSx core.

## Summary of PRs:

- [x] Remove Proposals from Executions, Transfer entities and add deterministic IDs 
- [x] Address linting issues 
- [x] Remove Plugin data from the Subgraph ${\color{Red}<-- This PR}$ 
- [x] Rename TransactionActions back to Actions ${\color{Red}<-- This PR}$
- [ ] Move ID generation to OSx Commons if needed

## This PR

Removes all plugin data from the OSx core subgraphs:

- Renamed Proposal to `TransactionActions` and made it immutable
- Added deterministicID as field to `TransactionActions` and `Action`
- `Action` swaps `proposal` foreign key in place of `transactionAction`
- The `deterministicId` and `Id` fields now include the `daoAddress` inside executeActions
- Renamed `proposal` to `transactionActions` in ERC20Transfer, ERC721Transfer, ERC1155Transfer and NativeTransfer and DAO
- Removed `startDate` and `endDate` from `TransactionActions`

- Removed `TokenVoting` `Multisig` `Admin` and `AddressList` from the schema
- Deleted the corresponding handlers and references in code
- Removed the concept of wrapped ERC20 as these are only relevant in the context of tokenVoting
- identifyAndFetchOrCreateERC20Entity is just `fetchOrCreateERC20Entity`

Task ID: [OS-1228](https://aragonassociation.atlassian.net/browse/OS-1228)

## Type of change

- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [x] I have updated the `CHANGELOG.md` file in the root folder.
- [ ] I have updated the `DEPLOYMENT_CHECKLIST` file in the root folder.
- [ ] I have updated the `UPDATE_CHECKLIST` file in the root folder.
- [ ] I have updated the Subgraph and added a QA URL to the description of this PR.

**Subgraph Link**
https://subgraph.satsuma-prod.com/aragon/osx-polygon-jordan-polygon/version/split-action-batch/playground

[OS-1228]: https://aragonassociation.atlassian.net/browse/OS-1228?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/subgraph/CHANGELOG.md [hunk 1]
```diff
@@ -10,12 +10,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- `Action` swaps `proposal` foreign key in place of `transactionAction`
 - The `deterministicId` and `Id` fields now include the `daoAddress` inside executeActions
-- Renamed `proposal` to `transactionActions` in ERC20Transfer, ERC721Transfer, ERC1155Transfer and NativeTransfer
+- Renamed `proposal` to `transactionActions` in ERC20Transfer, ERC721Transfer, ERC1155Transfer and NativeTransfer and DAO
 - Removed `startDate` and `endDate` from `TransactionActions`
 
+- Removed `TokenVoting` `Multisig` `Admin` and `AddressList` from the schema
+- Deleted the corresponding handlers and references in code
+- Removed the concept of wrapped ERC20 as these are only relevant in the context of tokenVoting
+- identifyAndFetchOrCreateERC20Entity is just `fetchOrCreateERC20Entity`
```
Comment (Michael-A-Heuer) [inline packages/subgraph/CHANGELOG.md:22]: Can we write this better? I.e., format variables as code and specify where this was changed for traceability?

**Review by Michael-A-Heuer — CHANGES_REQUESTED** [review]
Please make the `CHANGELOG` a bit clearer so that we can easily understand which changes have happened in 6 months from now

**Review by novaknole — APPROVED** [review]
Very good.

File: packages/subgraph/CHANGELOG.md [hunk 2]
```diff
@@ -9,12 +9,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Split
 
-- Renamed Proposal to `TransactionActions` and made it immutable
-- Added deterministicID as field to `TransactionActions` and `TransactionAction`
-- Temp immutable entity `TransactionAction` created while plugins (with actions) still live inside OSx. Includes `deterministicId` and swaps `proposal` foreign key in place of `transactionAction`
+- Renamed Proposal to `ActionBatch` and made it immutable
```
Comment (Michael-A-Heuer) [inline packages/subgraph/CHANGELOG.md:12]:
```suggestion
- Renamed `Proposal` to `ActionBatch` and made it immutable
```

File: packages/subgraph/CHANGELOG.md [hunk 3]
```diff
@@ -9,12 +9,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Added `deterministicId` as field to `ActionBatch` and `Action`. The `deterministicId` is a reference shared by the ActionId in plugins packages that can be known at the time of proposal execution.
 - The `deterministicId` and `Id` fields now include the `daoAddress` inside executeActions
-- Renamed `proposal` to `transactionActions` in ERC20Transfer, ERC721Transfer, ERC1155Transfer and NativeTransfer
-- Removed `startDate` and `endDate` from `TransactionActions`
+- Renamed `proposal` to `actionBatch` in `Action`, `ERC20Transfer`, `ERC721Transfer`, `ERC1155Transfer`, `NativeTransfer` and `DAO`
```
Comment (Michael-A-Heuer) [inline packages/subgraph/CHANGELOG.md:15]:
```suggestion
- Renamed `Proposal` to `actionBatch` in `Action`, `ERC20Transfer`, `ERC721Transfer`, `ERC1155Transfer`, `NativeTransfer` and `DAO`
```

File: packages/subgraph/CHANGELOG.md [hunk 4]
```diff
@@ -9,12 +9,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Split
 
-- Renamed Proposal to `TransactionActions` and made it immutable
-- Added deterministicID as field to `TransactionActions` and `TransactionAction`
-- Temp immutable entity `TransactionAction` created while plugins (with actions) still live inside OSx. Includes `deterministicId` and swaps `proposal` foreign key in place of `transactionAction`
+- Renamed Proposal to `ActionBatch` and made it immutable
+- Added `deterministicId` as field to `ActionBatch` and `Action`. The `deterministicId` is a reference shared by the ActionId in plugins packages that can be known at the time of proposal execution.
 - The `deterministicId` and `Id` fields now include the `daoAddress` inside executeActions
```
Comment (Michael-A-Heuer) [inline packages/subgraph/CHANGELOG.md:14]:
```suggestion
- The `deterministicId` and `Id` fields now include the `daoAddress` inside `executeActions`
```

File: packages/subgraph/CHANGELOG.md [hunk 5]
```diff
@@ -9,12 +9,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
-- Renamed `proposal` to `transactionActions` in ERC20Transfer, ERC721Transfer, ERC1155Transfer and NativeTransfer
-- Removed `startDate` and `endDate` from `TransactionActions`
+- Renamed `proposal` to `actionBatch` in `Action`, `ERC20Transfer`, `ERC721Transfer`, `ERC1155Transfer`, `NativeTransfer` and `DAO`
+- Removed `startDate` and `endDate` from `ActionBatch` as they relate to proposals
+- Removed `TokenVoting` `Multisig` `Admin` and `AddressList` from the schema
+- Deleted the corresponding handlers any references to the above plugins
+- Removed the concept of wrapped ERC20 as these are only relevant in the context of tokenVoting
```
Comment (Michael-A-Heuer) [inline packages/subgraph/CHANGELOG.md:19]:
```suggestion
- Removed the concept of wrapped ERC20 as these are only relevant in the context of the token voting plugin
```

File: packages/subgraph/CHANGELOG.md (hunk 3 above)
Comment (jordaniza) [reply packages/subgraph/CHANGELOG.md:15]: Deliberately ignored, the original is correct

**Review by Michael-A-Heuer — APPROVED** [review]

**Pull request merged**